src/main/java/seedu/address/model/quiz/Quiz.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.model.quiz;
+
+import seedu.address.model.flashcard.Answer;
+import seedu.address.model.flashcard.FlashcardSet;
+import seedu.address.model.flashcard.Question;
+
+public class Quiz {
+
+    private final FlashcardSet flashcardSet;
+    private final int flashcardSetIndex;
```

Comment:
Could you help check whether the Index class would be useful in this case instead of int? The flashcard class is implemented using Index. TLDR: the Index class adds flexibility of zeroBasedIndex and oneBasedIndex. I'm just not sure whether it's actually useful, need second eye. Thanks!! 👍